bindings/py/src/plugin/PyBindRegion.hpp

```diff
@@ -169,18 +149,19 @@ namespace nupic
         std::string className_;
 
         pybind11::object node_;
-  		std::set<std::shared_ptr<PyArray<UInt64>>> splitterMaps_;
-        
+  		//std::set<std::shared_ptr<PyArray<UInt64>>> splitterMaps_;
+
         // pointers rather than objects because Array doesnt
         // have a default constructor
         std::map<std::string, Array*> inputArrays_;
```

Comment:
> PyBindRegion.hpp
(smart)_ptr instead of Arr* ?

don't understand why this was a pointer.  The Array class is just a wrapper around a smart pointer.  There is no reason to not put it directly into the map.

Reply:
We should probably put it off for later, but that would be something to see if it can be done right.